### ISSUE TITLE
Ablation experiments of masking pre-training with multiple datasets

### ISSUE THREAD

**Issue by shmh40** [ecmwf/WeatherGenerator#764]
Experiments with different masking strategies on multiple datasets (ERA5+CERRA, ERA5+CERRA+OPERA+SEVIRI) to train skilful model for hackathon. Evaluation of 0-shot forecast skill and forecast skill after fine-tuning.
